lerna.json

```diff
@@ -3,13 +3,16 @@
   "version": "independent",
   "npmClient": "yarn",
   "useWorkspaces": true,
```

Comment:
I think as a part of this request, we can also raise the version of Lerna. As #3899 probably won't be merged in the near future. 
Upgrade to v6 is non-breaking. From v6 to v7 commands: *add*, *link*, *bootstrap* are dropped. 
and *useWorkspaces* too. Running *lerna repair* on v7 should remove this flag. [Here is the changelog for v7](https://github.com/lerna/lerna/blob/main/CHANGELOG.md#700-2023-06-08)

Reply:
It also include some fixes for publish package, which is good for us